src/components/entity/EntityResult.vue

```diff
@@ -216,10 +216,22 @@
                   'justify-center': !$vuetify.breakpoint.lgAndUp,
                 }"
               >
-                <v-btn icon @click="incRelationshipStartIndex(-1)">
+                <v-btn
+                  :class="{ hidden: relationshipStartIndex <= 0 }"
```

Comment:
Instead of hiding, just disable them so its consistent with the buttons on the search page

Reply:
ya, that sounds cleaner.

Change completed